src/test/resources/docker/SimpleGradle_7_6.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gradle:7.6-jdk11
+
+# Do not change SRC_DIR, value is expected by tests
+ENV SRC_DIR=/opt/project/src
+
+ENV JAVA_TOOL_OPTIONS="-Dhttps.protocols=TLSv1.2"
+
+
+# Install git
+RUN apt-get update && apt-get install -y git
+
+# Set up the test project
+RUN mkdir -p ${SRC_DIR}
+
+
+RUN git clone --depth 1 https://github.com/jabedhasan21/java-hello-world-with-gradle ${SRC_DIR}
```

Comment:
What happens when the employee leaves the company? We shouldn't be using personal projects.

Reply:
This is just a sample project publicly available on github.

Reply:
But it's tied to an individual account. Once someone leaves the company, he/she can make it a private project or just delete the whole project. In that case, the build starts failing. 
I know it's a very simple project but I feel it should be under an organization and not tied to any individual. Not a good practice.

Reply:
Also, if the individual becomes a bad actor or the account is hacked then one can introduce a malicious code through `gradle build` in our CI server which can affect a lot of different products inside Synopsys VPN.